docker-multinode/README.md

```diff
@@ -17,7 +16,7 @@ interested in just starting to explore Kubernetes, we recommend that you start t
 
 ## Prerequisites
 
-The only thing you need is a machine with **Docker 1.7.1 or higher**
+The only thing you need is a machine with **Docker 1.8.3 or higher**
```

Comment:
Why is 1.8.3 now necessary? It is just for docker daemon vs docker -d? 1.7.1 is the newest version CentOS 6 will ever get, and I got the previous scripts to successfully work with it.